qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -224,6 +379,35 @@ def __init__(self, parent: Instrument, name: str, channel: str) -> None:
         self.add_parameter('fastsweep',
                            parameter_class=LuaSweepParameter)
 
+        self.add_parameter('npts',
+                           initial_value=500,
+                           label='Number of points',
+                           get_cmd=None,
+                           set_cmd=None)
+
+        self.add_parameter('dt',
+                           initial_value=1e-3,
+                           label='Time resolution',
+                           unit='s',
+                           get_cmd=None,
+                           set_cmd=None)
+
+        self.add_parameter(name='time_axis',
+                           label='Time',
+                           unit='s',
+                           vals=vals.Arrays(shape=(self.npts,)),
+                           parameter_class=TimeAxis)
+
+        self.add_parameter('timetrace',
+                           vals=vals.Arrays(shape=(self.npts,)),
+                           setpoints=(self.time_axis,),
+                           parameter_class=TimeTrace)
+
+        self.add_parameter('timetrace_mode',
+                           get_cmd = None,
+                           set_cmd = self.timetrace._set_mode,
+                           vals=vals.Enum('i', 'v'))
```

Comment:
There are different schools of thought on this topic, but I prefer elaborate, easy-to-read names over quickly typed names. Should we change `"i"` and `"v"` to `"current"` and `"voltage"`?